front/app/api/idea_statuses/useAddIdeaStatus.ts

```diff
@@ -10,7 +10,7 @@ const addIdeaStatus = async (requestBody: IIdeaStatusAdd) =>
   fetcher<IIdeaStatus>({
     path: '/idea_statuses',
     action: 'post',
-    body: requestBody,
+    body: { idea_status: requestBody },
   });
```

Comment:
@IvaKop Just want to double-check that we want this nested under `idea_status` like this? :)